cirq/linalg/decompositions.py

```diff
@@ -180,18 +169,14 @@ def kron_factor_4x4_to_2x2s(
         matrix: np.ndarray,
 ) -> Tuple[complex, np.ndarray, np.ndarray]:
     """Splits a 4x4 matrix U = kron(A, B) into A, B, and a global factor.
-
     Requires the matrix to be the kronecker product of two 2x2 unitaries.
     Requires the matrix to have a non-zero determinant.
     Giving an incorrect matrix will cause garbage output.
-
```

Comment:
Don't remove these newlines. There should be a newline separating the sections of the documentation.

Reply:
That wasn't intended and honestly I'm not sure how I did it but I'll run through the files and undo it.  My primary concern with having (-pi/4+atol, +pi/4+atol] is that you are explicitly allowing two sets of coefficients which are equivalent up to local operations to exist in the canonical region. This seems like it could break code that tries to convert two operations that are equivalent up to local operations into each other. Granted, my choice to exclude x in (pi/4-atol, pi/4) and z<0 may cause problems in a similar way. My hope is that the test I posted provides evidence that this isn't the case, since operations that start in that region are tested.